_posts/2022-11-29-November-Update-Exploring-V3D.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "November Update: Exploring V3D"
+date: 2022-11-28
+author: Maíra Canal
+layout: post
+permalink: /november-update-exploring-v3d/
+categories: Genel
+tags: [igalia, graphics, kernel]
+---
+
+After I posted my blog post, it has been a busy couple of months. As I pointed on my last blog post after I finished GSoC, I joined the [Igalia Coding Experience](https://www.igalia.com/coding-experience/) mentorship project, and also, in October, I traveled to Minneapolis for XDC 2022. So, let's make a summary of the last couple of months.
```

Comment:
- Better start in: It has been a busy couple of months. As I pointed on my last blog post, I finished GSoC and joined the Igalia Coding Experience mentorship project. In October, I also travelled to Minneapolis for XDC 2022 where I presented to the Linux Graphics community our AMD/KUnit work with my colleagues. So, let's make a summary of the last couple of months.